src/components/AllScheduledTransactions.js

```diff
@@ -0,0 +1,80 @@
+import React from 'react'
+
+import API from '../API';
+import { CASH_ACCOUNT_LIST_URL } from '../Config';
+
+import ModalComponent from './Modals';
+import ScheduleTransactionForm from './Transactions/ScheduleTransactionForm.js';
+import ScheduledTransactionList from './Transactions/ScheduledTransactionList/scheduledTransactionList';
+
+
+
+
+class AllScheduledTransactions extends React.Component {
+
+    constructor(props) {
+        super(props);
+        this.state = {
+            isLoaded: false,
+            categories: [],
+            cashAccounts: []
+        }
+    }
+
+
+    componentDidMount() {
+        (async () => {
+            const cashAccounts = await API.fetchCashAccountList(CASH_ACCOUNT_LIST_URL + '?page_size=20')
```

Comment:
why is the page size hardcoded here? I thought it was dynamic and would change with data table